platform/android/export/export.cpp

```diff
@@ -768,6 +768,81 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		}
 	}
 
+	void _write_tmp_manifest(const Ref<EditorExportPreset> &p_preset, bool p_give_internet, bool p_debug) {
+		String manifest_text =
+				"<?xml version=\"1.0\" encoding=\"utf-8\"?>\n"
```

Comment:
Throughout this method, the use of multiline string literals should make it easier to parse the added manifest content since you'd be able to omit the escape characters.

Reply:
In some circumstances t's really difficult to use string literals (`R"(...)"` strings) with clang-format. Do you have any workarounds? Apparently I'm [not the only one](https://github.com/rism-ch/verovio/issues/387) with this issue. The solution that forum gives is to change the clang-format config file, but I'm not sure if that's the best idea.

Reply:
If that's an issue, then nevermind. With your updates the code is a lot more readable, so having the strings easily readable is a nice to have at this point, not a must. So let's keep the current string format if that's easier.